docs/references/config/config.toml.md

```diff
@@ -0,0 +1,913 @@
+runs on.
+
+### rpc.max_subscription_clients
+Maximum number of unique clientIDs that can subscribe to events at the `/subscribe` RPC endpoint.
+```toml
+max_subscription_clients = 100
+```
+
+| Value type          | integer |
+|:--------------------|:--------|
+| **Possible values** | &gt;= 0 |
+
+If you are using the `/broadcast_tx_commit` RPC endpoint, set this to the estimated maximum number of
+`broadcast_tx_commit` calls per block.
+
+### rpc.max_subscriptions_per_client
+Maximum number of unique queries a given client can subscribe to at the `/subscribe` RPC endpoint.
+```toml
+max_subscriptions_per_client = 5
+```
+
+| Value type          | integer |
+|:--------------------|:--------|
+| **Possible values** | &gt;= 0 |
+
+If you are using the `/broadcast_tx_commit` RPC endpoint, set this to the estimated maximum number of
+`broadcast_tx_commit` calls per block.
+
+### rpc.experimental_subscription_buffer_size
+Experimental parameter to specify the maximum number of events a node will buffer, per subscription, before returning
+an error and closing the subscription.
+```toml
+experimental_subscription_buffer_size = 200
+```
+
+| Value type          | integer   |
+|:--------------------|:----------|
+| **Possible values** | &gt;= 100 |
+
+Higher values will accommodate higher event throughput rates (and will use more memory).
+
+### rpc.experimental_websocket_write_buffer_size
+Experimental parameter to specify the maximum number of RPC responses that can be buffered per WebSocket client.
+```toml
+experimental_websocket_write_buffer_size = 200
+```
+
+| Value type          | integer                                         |
+|:--------------------|:------------------------------------------------|
+| **Possible values** | &gt;= rpc.experimental_subscription_buffer_size |
+
+
+If clients cannot read from the WebSocket endpoint fast enough, they will be disconnected, so increasing this parameter
+may reduce the chances of them being disconnected (but will cause the node to use more memory).
+
+If set lower than `rpc.experimental_subscription_buffer_size`, connections could be dropped unnecessarily. This value
+should ideally be somewhat higher to accommodate non-subscription-related RPC responses.
+
+### rpc.experimental_close_on_slow_client
+Close the WebSocket client in case it cannot read events fast enough. Allows greater predictability in subscription
+behaviour.
+```toml
+experimental_close_on_slow_client = false
+```
+
+| Value type          | boolean |
+|:--------------------|:--------|
+| **Possible values** | `false` |
+|                     | `true`  |
+
+The default behaviour for WebSocket clients is to silently drop events, if they cannot read them fast enough. This does
+not cause an error and creates unpredictability.
+Enabling this setting creates a predictable outcome by closing the WebSocket connection in case it cannot read events
+fast enough.
+
+### rpc.timeout_broadcast_tx_commit
```

Comment:
I would warn users to not use this broadcast method in production networks. As the implementation is _super_ inefficient.

Reply:
ad993d5f2